src/gc/config.d

```diff
@@ -26,7 +26,7 @@ struct Config
 {
     bool disable;            // start disabled
     ubyte profile;           // enable profiling with summary when terminating program
-    string gc = "conservative"; // select gc implementation conservative|manual
+    string gc = "conservative"; // select gc implementation conservative|precise|manual
```

Comment:
Please enable the precise GC for running the new code on the auto testers. It should be changed back to conservative before merging, though.